spec/unit/resource/api1800/c7000/lig_uplink_set.rb

```diff
@@ -0,0 +1,20 @@
+# (C) Copyright 2020 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1800::C7000::LIGUplinkSet do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API600::C7000::LIGUplinkSet' do
+    expect(described_class).to be < OneviewSDK::API600::C7000::LIGUplinkSet
```

Comment:
Please change it to API1600 and change in description also

Reply:
specs are written only till API600.. so inherited from that file